wcomponents-theme/src/main/xslt/wc.ui.table.xsl

```diff
@@ -929,6 +929,8 @@
 		<xsl:value-of select="@minSelectedRows"/>
 		<xsl:text>","max":"</xsl:text>
 		<xsl:value-of select="@maxSelectedRows"/>
+		<xsl:text>","otherSelected":"</xsl:text>
+		<xsl:value-of select="@selectedOnOther"/>
```

Comment:
This will create generate something like this: `{ "otherSelected": "12" }`
You have based this on existing examples, like `max` and `min` but actually do we need all those double quotes? What do you think?

Reply:
Probably not, I just like consistency and assumed that you guys had done it that way for a reason.

Reply:
Could this be impacting your addition operation (above)? If you are reading this property directly then you will have a String rather than a Number on at least one side of the `+`.

Reply:
Yep, that was it.